### PR TITLE
Adyen: Pass phone, statement, device_fingerprint

### PR DESCRIPTION
**Pull request by curiousepic**
Remote:
51 tests, 147 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

Unit:
30 tests, 144 assertions, 0 failures, 0 errors, 0 pendings, 0 omissions, 0 notifications
100% passed

### CODE REVIEW

File: test/remote/gateways/remote_adyen_test.rb [hunk 1]
```diff
@@ -140,7 +140,7 @@ def test_successful_purchase_no_cvv
   end
 
   def test_successful_purchase_with_more_options
-    options = @options.merge!(fraudOffset: '1', installments: 2)
+    options = @options.merge!(fraudOffset: '1', installments: 2, shopper_statement: 'statement note', device_fingerprint: 'm7Cmrf++0cW4P6XfF7m/rA')
```
Comment (jknipp) [inline test/remote/gateways/remote_adyen_test.rb]: The change looks good, I notice that we don't specifically test `telephoneNumber`, but we are testing the other new fields. I just want to make sure that was intentional.

Comment (curiousepic) [reply test/remote/gateways/remote_adyen_test.rb]: It was intention in the sense that it's a common field we don't normally test for anyhow, and now all other tests nominally pass the default phone generated by the `credit_card` creation helper.

Comment (jknipp) [reply test/remote/gateways/remote_adyen_test.rb]: 👍 Works for me.

**Comment by curiousepic** [comment]
Closed with 90a551572ddd4c60bcf6b2f34ede750fd131fc8f

**Pull request closed without merging**